pom.xml

```diff
@@ -97,6 +97,13 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-test</artifactId>
             <version>4.0.5.RELEASE</version>
+
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-test</artifactId>
+            <version>5.2.6.RELEASE</version>
```

Comment:
this new entry looks like it replaces the one above. If so I think we can remove the old one?